opentelemetry-exporter-gcp-monitoring/src/opentelemetry/exporter/cloud_monitoring/_resource.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import Dict, Optional, Tuple
+
+from google.api.monitored_resource_pb2 import MonitoredResource
+from opentelemetry.sdk.resources import Attributes, Resource
+
+
+# TODO: use opentelemetry-semantic-conventions package for these constants once it has
+# stabilized. Right now, pinning an unstable version would cause dependency conflicts for
+# users so these are copied in.
+class ResourceAttributes:
+    AWS_EC2 = "aws_ec2"
+    CLOUD_ACCOUNT_ID = "cloud.account.id"
+    CLOUD_AVAILABILITY_ZONE = "cloud.availability_zone"
+    CLOUD_PLATFORM_KEY = "cloud.platform"
+    CLOUD_PROVIDER = "cloud.provider"
+    CLOUD_REGION = "cloud.region"
+    GCP_COMPUTE_ENGINE = "gcp_compute_engine"
+    GCP_KUBERNETES_ENGINE = "gcp_kubernetes_engine"
+    HOST_ID = "host.id"
```

Comment:
Did you see comment on L22? Unfortunately each release of the semconv package can have breaking changes (https://github.com/open-telemetry/opentelemetry-python/issues/2946). I don't want to create dependency conflicts for users if I pin the version

Reply:
ah, missed the TODO.  I don't think the entire conventions package could ever become stable, though, since there might always be experimental/unstable attributes present.

Reply:
Ya, I'm in favor of the https://github.com/open-telemetry/opentelemetry-python/issues/2946 approach (which Go SIG does) but some folks were opposed to it